docs/README.md

```diff
@@ -39,6 +39,7 @@ The `add-to-cart-button` is a block responsible for adding products in the [Mini
 
 | Prop name               | Type      | Description                                                                       | Default value        |
 | ----------------------- | --------- | --------------------------------------------------------------------------------- | -------------------- |
+| `onClickBehavior`       | `string` | Whether the user should be redirected to the product page but the product will not be added to cart (`go-to-product-page`) | `add-to-cart`              |
```

Comment:
It would be better to type this as `enum` instead of `string` since this prop only has two possible values.

```suggestion
| `onClickBehavior`       | `enum` | Controls what happens when users click the button. Possible values are: `go-to-product-page` or `add-to-cart`. | `add-to-cart`              |
```